R/download_dataset.R

```diff
@@ -0,0 +1,170 @@
+#' Download data for publication
+#'
+#' ...
+#'
+#' @param connection A valid connection to the ETN database.
+#' @param animal_project_code (string) Animal project to download data from.
```

Comment:
It's common practice to add default value while documenting args. It seems we have never done before, but I think this is a good function to start with. Especially in this function is quite important because if no animal project is assigned (`animal_project_code = NULL`), ALL data are download, and that's quite a big difference in terms of file dimensions, I think.

I would add the following:
Default: `NULL`. If `NULL`, the data from all animal projects in etn database are downloaded.

Reply:
I would not download all data by default! animal_project_code should be a required parameter, not sure how to do that consistently. Probably by not adding `= NULL`?

Reply:
Yes, indeed. REmoving `= NULL` you make that argument mandatory for user.

Reply:
I have changed this to not adding `=NULL` with a check on `is.null()` and `missing()`. It is now also documented what the defaults are, but only for this function.

Reply:
See e7cb43b93e09a37fa0c879e51cd915e6e5dedbdd I have now done this for all functions.